### PR TITLE
Solution and project updates

### PR DESCRIPTION
**Pull request by Hyrtwol**
Hey, wanted to learn dx12 and I have used SharpDX before, so ended up here. I really liked the good list os samples.
Ended up fixing the solution and csproj files so it's a bit easier to deal with and thought I should share ;)
I know SharpDX have men stale for a while now but still a good learning resource.
I wonder how difficult it would be to swap SharpDX out with e.g. https://github.com/amerkoleci/Vortice.Windows/

Cheers T

### CODE REVIEW

File: Samples/20-Shadows/ShadowsApp.cs [hunk 1]
```diff
@@ -244,7 +244,7 @@ protected override void Draw(GameTimer gt)
             CommandQueue.ExecuteCommandList(CommandList);
 
             // Present the buffer to the screen. Presenting will automatically swap the back and front buffers.
-            SwapChain.Present(0, PresentFlags.None);
+            SwapChain.Present(1, PresentFlags.None);
```
Comment (discosultan) [inline Samples/20-Shadows/ShadowsApp.cs:247]: What's the motivation behind this change?

**Review by discosultan — COMMENTED** [review]
Great effort, thank you! Will merge as soon as comments are resolved.

> I wonder how difficult it would be to swap SharpDX out with e.g. https://github.com/amerkoleci/Vortice.Windows/

I don't know enough about both projects to assess that. However, if there is a strong reason to do so, I'd be happy to accept a PR. Or perhaps the Vortice version could live as a fork or a standalone repo.

**Comment by discosultan** [comment]
@Hyrtwol Went ahead and merged it. Thanks again for the contribution!

**Comment by Hyrtwol** [comment]
Hey nice, sry for not answering reg. the present thing i just forgot to remove my gfx card was burning hot :p

File: Samples/20-Shadows/ShadowsApp.cs (hunk 1 above)
Comment (Hyrtwol) [reply Samples/20-Shadows/ShadowsApp.cs:247]: Just revert this ;)
